parameters/src/canary/mod.rs

```diff
@@ -19,6 +19,7 @@ pub use genesis::*;
 /// The restrictions list as a JSON-compatible string.
 pub const RESTRICTIONS_LIST: &str = include_str!("./resources/restrictions.json");
 
+#[cfg(not(feature = "cosmwasm"))]
 const REMOTE_URL: &str = "https://parameters.provable.com/canary";
```

Comment:
I'm curious why do we put a `const` under a (negative) feature flag.
(no issues, just curious)

Reply:
To avoid this message:

```
warning: constant `REMOTE_URL` is never used
  --> parameters/src/canary/mod.rs:22:7
   |
22 | const REMOTE_URL: &str = "https://parameters.provable.com/canary";
   |       ^^^^^^^^^^
   |
   = note: `#[warn(dead_code)]` (part of `#[warn(unused)]`) on by default
```